packages/jobs/src/job-definitions/batchEvaluations/runEvaluationJob.ts

```diff
@@ -64,7 +64,6 @@ export const runEvaluationJob = async (job: Job<RunEvaluationJobData>) => {
           documentUuid,
           status: finished ? 'finished' : 'running',
           ...progress,
-          completed: progress.completed < 1 ? 1 : progress.completed,
```

Comment:
Why this?

Reply:
it's a frontend concern

Reply:
And did you handle in frontend?

Reply:
i don't think it's particularly important to show 1/10 vs 0/10

Reply:
Ok then